os-autoinst-obs-auto-submit

```diff
@@ -42,6 +44,27 @@ update_package() {
     else
         $osc ci -m "Offline generation of ${version}"
     fi
+    local osc_query="https://api.opensuse.org/public/build/$dst_project/_result?repository=$submit_target&package=$package"
+    local wip_states='\(unknown\|blocked\|scheduled\|dispatching\|building\|signing\|finished\)'
+    local bad_states='\(failed\|unresolvable\|broken\)'
+    local attempts="$osc_build_start_poll_tries"
+    while ! curl -s  "$osc_query" | grep -e "$wip_states" ; do
+        if [[ $attempts -le 0 ]]; then
+            echo "warning: Re-build of $package has not been considered in time (or package has been re-built so fast that we've missed it)"
+            break
+        fi
+        echo "Waiting for re-build of $package to be considered ($attempts)"
```

Comment:
the attempts within parentheses will just appear as a bare number. I would make that explicit and write something like "attempts left: $attempts"